lib/cn/csched_sp3.c

```diff
@@ -2756,12 +2748,11 @@ sp3_create(
 
     memset(sp, 0, alloc_sz);
     sp->rp = rp;
+    sp->sp_healthy = true;
+    sp->sp_sval_min = THROTTLE_SENSOR_SCALE / 2;
     sp->health = health;
     sp->kvdb_alias = kvdb_alias;
 
-    mutex_init(&sp->mon_lock);
```

Comment:
This was added in a bad rebase on my REST PR onto Greg's space amp work. Greg moved the initialization further down in the function

Reply:
Need the following to work correctly, right?  The last diff for compiling with -O1.

```
diff --git a/lib/cn/csched_sp3.c b/lib/cn/csched_sp3.c
index 35f0edf5..835c0be7 100644
--- a/lib/cn/csched_sp3.c
+++ b/lib/cn/csched_sp3.c
@@ -2759,12 +2759,10 @@ sp3_create(
     sp->health = health;
     sp->kvdb_alias = kvdb_alias;
 
-    mutex_init(&sp->mon_lock);
-    cv_init(&sp->mon_cv);
-
     INIT_LIST_HEAD(&sp->mon_tlist);
     INIT_LIST_HEAD(&sp->spn_alist);
     INIT_LIST_HEAD(&sp->spn_rlist);
+    sp->sp_healthy = true;
 
     for (size_t tx = 0; tx < NELEM(sp->rbt); tx++)
         sp->rbt[tx] = RB_ROOT;
diff --git a/tools/hsettp/utils.c b/tools/hsettp/utils.c
index 234f635b..46c62fa2 100644
--- a/tools/hsettp/utils.c
+++ b/tools/hsettp/utils.c
@@ -19,7 +19,7 @@
 merr_t
 flatten(cJSON *const in, const char *const prefix, cJSON *const out)
 {
-    merr_t err;
+    merr_t err = 0;
     char *tmp = NULL;
 
     INVARIANT(in);
```

Reply:
Also, the humanize arg doesn't work:

```
$ /opt/hse/bin/hsettp -s $(sudo jq -r '.socket.path' /mnt/kvdbs/kvdb1/kvdb.pid) kvs-cn-tree-get -f tab -u -p 0 kvs1
Request failed: tools/hsettp/hsettp.c:315: Bad message (74)
```

Reply:
Checking...